package/com.unity.formats.usd/Runtime/Scripts/IO/Skel/SkeletonImporter.cs

```diff
@@ -134,15 +138,20 @@ public static class SkeletonImporter {
     public static void BuildSkinnedMesh(string meshPath,
                                         string skelPath,
                                         SkeletonSample skeleton,
-                                        SkelBindingSample meshBinding,
+                                        UsdSkelSkinningQuery skinningQuery,
                                         GameObject go,
                                         PrimMap primMap,
                                         SceneImportOptions options) {
-      string[] joints = meshBinding.joints;
+      VtTokenArray jointsAttr = new VtTokenArray();
+      skinningQuery.GetJointOrder(jointsAttr);
```

Comment:
I believe this function can fail -- it might be worth checking the API to see if it returns a bool to indicate success.